### PR TITLE
Terminology: high-availability masters -> high-availability control plane

### PR DESCRIPTION
**Pull request by marians**
This PR updates the terminology of the page currently titles [Set up High-Availability Kubernetes Masters](https://kubernetes.io/docs/tasks/administer-cluster/highly-available-master/):

- Change `High-Availability Kubernetes Masters` to `High-Availability Control Plane`
- Change `Kubernetes master` to `Kubernetes control plane node`
- Change `master replica` to `control plane node`

According to the terminology change, the file is renamed, with an `aliases` entry for the old URI being added.

In addition, some minor grammar corrections are made and an outdated URL to etcd documentation is updated.

In https://github.com/kubernetes/website/issues/21621 it has been decided to use racially neutral language throughout the Kubernetes project. Replacing the term "master" with "control plane" has been decided some time ago.

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/highly-available-master.md [hunk 1]
```diff
@@ -28,68 +28,70 @@ This document describes how to use kube-up/down scripts to manage highly availab
...
 MULTIZONE=true KUBE_GCE_ZONE=europe-west1-b  ENABLE_ETCD_QUORUM_READS=true ./cluster/kube-up.sh
 ```
 
-Note that the commands above create a cluster with one master;
-however, you can add new master replicas to the cluster with subsequent commands.
+Note that the commands above create a cluster with one control plane node;
+however, you can add new control plane nodes to the cluster with subsequent commands.
 
-## Adding a new master replica
+## Adding a new control plane nodes
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/highly-available-master.md:52]:
```suggestion
## Adding a new control plane node
```

**Comment by tengqm** [comment]
lgtm

File: content/en/docs/tasks/administer-cluster/highly-available-master.md (hunk 1 above)
Comment (marians) [reply content/en/docs/tasks/administer-cluster/highly-available-master.md]: Thanks!

**Review by irvifa — COMMENTED** [review]
/approve

**Pull request merged**